cilium/cmd/cleanup.go

```diff
@@ -486,11 +486,14 @@ func getTCFilters(link netlink.Link) ([]*netlink.BpfFilter, error) {
 		}
 		for _, f := range filters {
 			if bpfFilter, ok := f.(*netlink.BpfFilter); ok {
+				// Filters created Go bpf loader are of format 'cilium-<iface>'.
+				// iproute2 would use the filename and section, e.g. bpf_overlay.o:[from-overlay].
 				if strings.Contains(bpfFilter.Name, "bpf_netdev") ||
```

Comment:
Nit: perhaps a regex match is simpler?
Naive attempt: `^bpf_(netdev|network|host|lxc|overlay)|cilium.*`
And then have a global `regex.Regexp` created with `regexp.MustCompile()`

Reply:
Good suggestion, but from a maintainability perspective, it really helps if those `bpf_*` entries to turn up in greps.